src/pages/LearningResources.js

```diff
@@ -1,231 +1,178 @@
-            <a
-              href='https://www.youtube.com/user/wesbos'
-              target='_blank'
-              rel='noopener noreferrer'
-            >
-              Wes Bos Youtube
-            </a>
-          </li>
-
-          <li className='underline mb-3 text-blue-700 hover:text-teal-600'>
-            <a
-              href='https://www.youtube.com/channel/UCyU5wkjgQYGRB0hIHMwm2Sg'
-              target='_blank'
-              rel='noopener noreferrer'
-            >
-              LevelUp Tutorials Youtube
-            </a>
-          </li>
-
-          <li className='underline mb-3 text-blue-700 hover:text-teal-600'>
-            <a
-              href='https://www.youtube.com/channel/UC-T8W79DN6PBnzomelvqJYw'
-              target='_blank'
-              rel='noopener noreferrer'
-            >
-              James Quick Youtube
-            </a>
-          </li>
-
-          <li className='underline mb-3 text-blue-700 hover:text-teal-600'>
-            <a
-              href='https://www.levelupfinancialplanning.com/videos/'
-              target='_blank'
-              rel='noopener noreferrer'
-            >
-              Level Up Financial Planning
-            </a>
-          </li>
-
-          <li className='underline mb-3 text-blue-700 hover:text-teal-600'>
-            <a
-              href='https://www.codecademy.com/'
-              target='_blank'
-              rel='noopener noreferrer'
-            >
-              Codecademy (Freemium)
-            </a>
-          </li>
-
-          <li className='underline mb-3 text-blue-700 hover:text-teal-600'>
-            <a
-              href='https://www.youtube.com/user/TechGuyWeb'
-              target='_blank'
-              rel='noopener noreferrer'
-            >
-              Traversy Media
-            </a>
-          </li>
-
-          <li className='underline mb-3 text-blue-700 hover:text-teal-600'>
-            <a
-              href='https://www.youtube.com/user/programmingwithmosh'
-              target='_blank'
-              rel='noopener noreferrer'
-            >
-              Programming With Mosh
-            </a>
-          </li>
-        </motion.ul>
+    <div className='container m-auto max-w-screen-xl align-middle mt-32 sm:max-w-screen-lg'>
+      <div class='relative px-12 text-center mx-auto my-10 sm:px-40 lg:px-32'>
+        <h2 class='text-2xl leading tracking-tight font-extrabold text-blue-900 sm:leading-10 sm:text-3xl'>
+          Lerning Resources
+        </h2>
+        <p className='text-base mb-2 text-blue-700 sm:mt-4 sm:px-12 sm:text-base md:text-base'> We've pulled some of the best free, and paid, resources together to help support your journey.
+        </p>
+        <p className='text-base text-blue-700 sm:mt-4 sm:text-base md:text-base'> Got a resource you think we should add to the list?{' '}
+          <a className='underline' href={'https://twitter.com/DevProtege'}>
```

Comment:
Let's make this a `<Link to='get-in-touch'>`

Reply:
This one has been addressed. 

Also added back in the Framer Motion div to match the other pages.